src/client/themes/cyberpunk/index.ts

```diff
@@ -69,8 +69,14 @@ const cyberpunkTheme: DefaultTheme = {
       primary: '#00A3FF',
       onPrimary: '#FFFFFF',
       loading: '#FFA800',
-      error: '#FF005E',
-      warning: '#FF7500',
+      error: {
+        background: '#FF005E',
```

Comment:
Is this the background-color? I think if so, we should try and use `backgroundColor` instead, unsure if that's a big refactor

Reply:
fixed on 43649ef98fa2d6e9d704f21f6f7892a3ba13f7f2